app/assets/stylesheets/_colors.scss

```diff
@@ -7,5 +7,8 @@ $off-white: #F9F9F9;
 $blue: #005e74;
 $light-blue: #CEE8F1;
 $lighter-blue: #EDFAFF;
-$error: #ad2000;
-$success: #256600;
+
+$action: #31b0d5;
+$error: #c9302c;
+$success: #449d44;
+$caution: orange;
```

Comment:
I like how you use hex colours for everything except caution. It's `orange`

Reply:
:skull_and_crossbones: